README.md

```diff
@@ -7,18 +7,23 @@ Our default implementation works with Amazon Web Services (AWS), but AWS Amplify
 - **API Documentation**
   https://docs.amplify.aws/start/q/integration/ios
 
-[![Discord](https://img.shields.io/discord/308323056592486420?logo=discord)](https://discord.gg/jWVbPfC) 
+[![Cocoapods](https://img.shields.io/cocoapods/v/Amplify)](https://cocoapods.org/pods/Amplify)
+[![CircleCI](https://circleci.com/gh/aws-amplify/amplify-ios.svg?style=shield)](https://circleci.com/gh/aws-amplify/amplify-ios)
+[![Discord](https://img.shields.io/discord/308323056592486420?logo=discord)](https://discord.gg/jWVbPfC)
 
 ## Features/APIs
 
-- [**Analytics**](https://aws-amplify.github.io/docs/ios/analytics): Easily collect analytics data for your app. Analytics data includes user sessions and other custom events that you want to track in your app.
-- [**API**](https://aws-amplify.github.io/docs/ios/api): Interact with your AWS AppSync API or make HTTP requests to your API Gateway endpoint with Amplify API. It provides a GraphQL client interface to use with Amplify Tool's model generation and automatic signing process to authenticate your requests.
-- [**Storage**](https://aws-amplify.github.io/docs/ios/storage): Provides a simple mechanism for managing user content for your app in guest, protected or private storage buckets.
-- [**Predictions**](https://aws-amplify.github.io/docs/ios/predictions): Provides a solution for using AI and ML cloud services to enhance your application.
+- [Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/ios) - for logging metrics and understanding your users.
+- [API (GraphQL)](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/ios) - for adding a GraphQL endpoint to your app.
+- [API (REST)](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/ios) - for adding a REST endpoint to your app.
+- [Authentication](https://docs.amplify.aws/lib/auth/getting-started/q/platform/ios) - for managing your users.
+- [DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/ios) - for making it easier to program for a distributed data store for offline and online scenarios.
+- [Predictions](https://docs.amplify.aws/lib/predictions/getting-started/q/platform/ios) - to detect text, images, and more!
+- [Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/ios) - store complex objects like pictures and videos to the cloud.
```

Comment:
Can we align the descriptions with Android? ie. copy paste from Android https://github.com/aws-amplify/amplify-android/blob/main/README.md then we can work on the wording in this comment if you want to adjust it, then apply to both for alignment 

- **[Authentication](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android)**
  APIs and building blocks for developers who want to create user authentication
  experiences.
- **[Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/android)**
  Provides a simple mechanism for managing user content for your app in public,
  protected or private storage buckets
- **[DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/android)**
  Provides a programming model for leveraging shared and distributed data
  without writing additional code for offline and online scenarios.
- **[API
  (GraphQL)](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/android)**
  Interact with your GraphQL server or AWS AppSync API with an easy-to-use &
  configured GraphQL client.
- **[API
  (REST)](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android)**
  Provides a simple solution when making HTTP requests. It provides an
  automatic, lightweight signing process which complies with AWS Signature
  Version 4.
- **[Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android)**
  Easily collect analytics data for your app. Analytics data includes user
  sessions and other custom events.
- **[Predictions](https://docs.amplify.aws/lib/predictions/getting-started/q/platform/android)**
  Connect your application with machine learning cloud services to enhance your
  application with natural language processing, computer vision, text to speech,
  and more.

Reply:
or we can take the current suggestions in the PR and update Android's readme

Reply:
I'm using the content from our docsite, which is aligned between Android and iOS. I'm happy to use Android's, as long as the docsite is updated as well.